core/src/main/java/com/linecorp/armeria/server/streaming/JsonTextSequences.java

```diff
@@ -149,6 +151,25 @@ public static HttpResponse fromPublisher(ResponseHeaders headers, Publisher<?> c
         return streamingFrom(contentPublisher, sanitizeHeaders(headers), trailers, o -> toHttpData(mapper, o));
     }
 
+    /**
+     * Creates a new JSON Text Sequences from the specified {@link Publisher}.
+     * Note that: This method is intentionally hidden from the public API for use only with
+     * {@code ScalaPbResponseConverterFunction} and {@code ProtobufResponseConverterFunction}.
+     *
+     * @param headers the HTTP headers supposed to send
+     * @param contentPublisher the {@link Publisher} which publishes the objects supposed to send as contents
+     * @param trailers the HTTP trailers
+     * @param contentConverter the function which converts the content object into a UTF-8 JSON string.
+     */
+    @SuppressWarnings("unused")
```

Comment:
How about adding some additional comments about why this is hidden from the public API and who calls this method?

Reply:
I'd added the reason to Javadoc. Is it better to add the comment in the code?